src/wlapplication_messages.cc

```diff
@@ -19,178 +19,273 @@
+		  "description of the AI training logic"),
+		false},
+	  {"", "auto_speed", "",
+		_("In multiplayer games only, this will keep adjusting the game speed "
+		  "automatically, depending on FPS. Useful in conjunction with --ai_training"),
+		false},
+	  /// Saving options
+	  {"", "autosave",
+		/** TRANSLATORS: A placeholder for a numerical value */
+		_("n"),
+		/** TRANSLATORS: `n` references a numerical placeholder */
+		_("Automatically save each `n` minutes"), false},
+	  {"", "rolling_autosave", _("n"),
+		/** TRANSLATORS: `n` references a numerical placeholder */
+		_("Use `n` files for rolling autosaves"), true},
+	  {"", "nozip", "", _("Do not save files as binary zip archives"), false},
+	  {"", "display_replay_filenames", _("[true*|false]"), _("Show filenames in replay screen"),
+		true},
+	  {"", "editor_player_menu_warn_too_many_players", _("[true*|false]"),
+		_("Enable verbose debug messages"), true},
+	  /// Game options
+	  {"", "auto_roadbuild_mode", _("[true*|false]"), _("Start building road after placing a flag"),
+		true},
+	  {"", "display_flags", _("[...]"), _("Display flags to set for new games"), true},
+#if 0  // TODO(matthiakl): Re-add training wheels code after v1.0
+	{"",
+	 "training_wheels",
+	 _("[true*|false]"),
+	 "",
+	 true
+	},
+#endif
+	  {"", "edge_scrolling",
+		/** TRANSLATORS: You may translate true/false, also as on/off or yes/no, but */
+		/** TRANSLATORS: it HAS TO BE CONSISTENT with the translation in the widelands textdomain. */
+		/** TRANSLATORS: * marks the default value */
+		_("[true|false*]"), _("Scroll when the mouse cursor is near the screen edge"), true},
+	  {"", "numpad_diagonalscrolling", _("[true|false*]"),
+		_("Allow diagonal scrolling with the numeric keypad"), true},
+	  {"", "game_clock", _("[true|false*]"), _("Display system time in the info panel"), true},
+	  {"", "ctrl_zoom", _("[true|false*]"), _("Zoom only when Ctrl is pressed"), true},
+	  {"", "single_watchwin", _("[true|false*]"), _("Use single watchwindow mode"), true},
+	  {"", "transparent_chat", _("[true*|false]"),
+		_("Show in-game chat with transparent background"), true},
+	  {"", "toolbar_pos", _("[...]"), _("Sets the toolbar location and mode"), true},
+	  /// Networking
+	  {_("Networking:"), "write_syncstreams",
+		/** TRANSLATORS: You may translate true/false, also as on/off or yes/no, but */
+		/** TRANSLATORS: it HAS TO BE CONSISTENT with the translation in the widelands textdomain. */
+		/** TRANSLATORS: * marks the default value */
+		_("[true*|false]"),
+		/** TRANSLATORS: A syncstream is a synchronization stream. Syncstreams are used in multiplayer
+		 */
+		/** TRANSLATORS: games to make sure that there is no mismatch between the players. */
+		_("Create syncstream dump files to help debug network games"), false},
+	  {"", "metaserver", _("URI"), _("Connect to a different metaserver for internet gaming"),
+		false},
+	  {"", "metaserverport", _("n"),
```

Comment:
missing translators comment

Reply:
One comment per unique string should be sufficient:
```
#. * TRANSLATORS: A placeholder for a numerical value
#: ../../src/wlapplication_messages.cc:94
#: ../../src/wlapplication_messages.cc:97
#: ../../src/wlapplication_messages.cc:144
#: ../../src/wlapplication_messages.cc:172
#: ../../src/wlapplication_messages.cc:179
#: ../../src/wlapplication_messages.cc:189
msgid "n"
msgstr ""
```